autoware_adapi_v1_msgs/package.xml

```diff
@@ -2,23 +2,29 @@
 <?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
 <package format="3">
   <name>autoware_adapi_v1_msgs</name>
-  <version>0.0.0</version>
-  <description>The autoware_adapi_v1_msgs package</description>
+  <version>1.1.0</version>
```

Comment:
The latest version is 1.2.0.
https://autowarefoundation.github.io/autoware-documentation/main/design/autoware-interfaces/ad-api/release/
```suggestion
  <version>1.2.0</version>
```